manifests/s/SimonG4ELI/SDR-Radio/Beta/3.4/SimonG4ELI.SDR-Radio.Beta.installer.yaml

```diff
@@ -1,48 +1,49 @@
-# Created with komac v2.13.0

```

Comment:
The diff of modifications made by this PR looks really weird now. Like the complete files content was removed and re-added (with minor modifications). Maybe squashing the two commits and/or re-basing against latest master could help?